subprojects/gradle/teamcity/build.gradle.kts

```diff
@@ -7,10 +7,11 @@ plugins {
 val teamcityRestClientVersion: String by project
 
 dependencies {
+    api("org.jetbrains.teamcity:teamcity-rest-client:$teamcityRestClientVersion")
```

Comment:
Is it time to move external deps to separate file?

Reply:
we can do it anytime

Reply:
In another task